"lardata

lgarren opened the pull request.

FNALbuild commented:
A new Pull Request was created by  @gartung (Patrick Gartung) for develop.

It involves the following packages:

lardata


@gartung, @FNALbuild can you please review it and eventually sign? Thanks.
@gartung this is something you requested to watch as well.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/larg4#2,LArSoft/larexamples#2,LArSoft/larpandora#2,LArSoft/larreco#2,LArSoft/larsim#2,LArSoft/larsoftobj#1,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/larg4#2,LArSoft/larexamples#2,LArSoft/larpandora#2,LArSoft/larreco#2,LArSoft/larsim#2,LArSoft/larsoftobj#1,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lardataobj#2,LArSoft/larevt#2,LArSoft/larevt#2,LArSoft/lareventdisplay#2,LArSoft/lareventdisplay#2,LArSoft/larexamples#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larpandora#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/larreco#2,LArSoft/larwirecell#2,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larsim#3,LArSoft/larsim#2,LArSoft/larreco#2,LArSoft/lareventdisplay#2,LArSoft/lardataalg#2,LArSoft/larana#2,LArSoft/larg4#2,LArSoft/larexamples#2,LArSoft/larwirecell#2,LArSoft/larcore#2,LArSoft/larevt#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/larpandora#2,LArSoft/lardataobj#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lareventdisplay#2,LArSoft/larevt#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larreco#2,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lareventdisplay#2,LArSoft/larevt#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larreco#2,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lareventdisplay#2,LArSoft/larevt#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larreco#2,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larwirecell#2

FNALbuild commented:
The tests are being triggered in jenkins.
Tested with other pull request(s) LArSoft/larsoft#2,LArSoft/larana#2,LArSoft/larcore#2,LArSoft/larcorealg#2,LArSoft/larcoreobj#2,LArSoft/lardataalg#2,LArSoft/lardataobj#2,LArSoft/lareventdisplay#2,LArSoft/larevt#2,LArSoft/larexamples#2,LArSoft/larg4#2,LArSoft/larpandora#2,LArSoft/larreco#2,LArSoft/larsim#2,LArSoft/larsim#3,LArSoft/larwirecell#2

FNALbuild commented:
+tests

gartung commented:
approved

FNALbuild commented:
This pull request is fully signed and it will be integrated in one of the next develop IBs (tests are also fine). This pull request will be automatically merged.

gartung commented:
code-checks

FNALbuild commented:
The code-checks are being triggered in jenkins.

FNALbuild commented:
+code-checks  
 for details see 
 https://buildmaster.fnal.gov/buildmaster/job/GithubIntegration/job/run-pr-code-checks//203/console

FNALbuild commented:
+tests

FNALbuild commented:
+tests

FNALbuild commented:
+tests

FNALbuild commented:
+tests

PetrilloAtWork committed:
Rewritten `lar::util::TrackPitchInView()`.

This is a candidate solution for issue #23817.
This also should make the function geometry-independent.

PetrilloAtWork committed:
Alignment and style update.

PetrilloAtWork committed:
Using default parameter type for RealComparisons.

PetrilloAtWork committed:
Added disabled code for the future come of `InterWireProjectedDistance()`.

PetrilloAtWork committed:
Merge remote-tracking branch 'origin/develop' into feature/gp_Issue23817

PetrilloAtWork committed:
Updated the disabled code to react to the new behaviour of `InterWireProjectedDistance()`.

lgarren committed:
whitespace fix

lgarren committed:
Merge pull request #1 from lgarren/feature/gp_Issue23817

Gianluca's fix for 23817

lgarren committed:
lardata v08_14_01 for larsoft v08_41_01

lgarren committed:
fix whitespace
